packages/backend/src/plugins/graphql.ts

```diff
@@ -24,5 +25,6 @@ export default async function createPlugin(
   return await createRouter({
     logger: env.logger,
     config: env.config,
+    modules: [Catalog],
```

Comment:
Looks like it should be pretty easy to migrate to the new backend system. I don't understand the second part, about catalog module. What do you mean?

Reply:
[Modules](https://backstage.io/docs/backend-system/architecture/modules) are an explicit concept in the new system, and typically they are closely tied to the plugin that they extend. In this case that may be a bit different though, since the catalog GraphQL module might really be closer tied to the catalog than to the GraphQL backend.

Reply:
It's a GraphQL module, not a backend module. What are profits of wrapping GraphQL module to a backend module, instead of just passing it directly?

Reply:
So that the GraphQL module implementation can get access to common services like logging or the catalog client